st7735r.h

```diff
@@ -15,7 +15,7 @@
 #define DISPLAY_SET_CURSOR_Y 0x2B
 #define DISPLAY_WRITE_PIXELS 0x2C
 
-#if defined(ST7789) || defined(ST7789VW)
+#if (defined(ST7789) || defined(ST7789VW)) && !defined(WAVESHARE_GAMEPI20)
```

Comment:
This #if won't be entered with the GamePi, and neither will the two #elifs below, so won't this cause the `#error Unknown display controller!` line be taken?

Can you change this to

```c
#if defined(WAVESHARE_GAMEPI20)
#define DISPLAY_NATIVE_WIDTH 240
#define DISPLAY_NATIVE_HEIGHT 320
#elif defined(ST7789) || defined(ST7789VW)
....
```
to follow the general flow of the size setup code here? (and drop the #define sizes in the changed block below?)